test/jasmine/tests/parcoords_test.js

```diff
@@ -868,7 +871,65 @@ describe('parcoords basic use', function() {
 
     });
 
-    it('@gl Calling `Plotly.restyle` with a string path should amend the preexisting parcoords', function(done) {
+    function _getGrayRatio(secondPass, msg)
+        var canvases = d3.selectAll('.gl-canvas');
+        expect(canvases.size()).toBe(3, msg);
+        
+        canvases.each(function(element, index) {
```

Comment:
... but alternatively, you could select `.gl-canvas-context` instead of all three `<canvas>` elements